packages/commonwealth/package.json

```diff
@@ -57,6 +57,7 @@
     "test-suite": "NODE_ENV=test nyc ts-mocha --project tsconfig.json './test/**/*.spec.ts'",
     "unit-test": "NODE_ENV=test ts-mocha --project tsconfig.json './test/unit/**/*.spec.ts'",
     "unit-test:watch": "NODE_ENV=test ts-mocha --project tsconfig.json --opts test/mocha-dev.opts './test/unit/**/*.spec.ts' -w --watch-files '**/*.ts'",
+    "unit-test-requirements:watch": "NODE_ENV=test ts-mocha --project tsconfig.json --opts test/mocha-dev.opts './test/unit/requirementsModule/*.spec.ts' -w --watch-files '**/*.ts'",
```

Comment:
should add docs for this in the knowledgebase

Reply:
tbh, I intend to delete this– only used it for convenience since running all the unit tests is oddly slow